src/components/List/ListItem/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import useStyles from './styles';
+import {
+  View,
+  Pressable,
+  StyleProp,
+  ViewStyle,
+  GestureResponderEvent,
+} from 'react-native';
+
+interface IListItemProps {
+  leftContent?: React.ReactNode;
+  rightContent?: React.ReactNode;
+  onPress?: (event: GestureResponderEvent) => void;
+  size?: 'small' | 'medium' | 'large';
+  divider?: boolean;
+  disabled?: boolean;
+  selected?: boolean;
+  testID?: string;
+  style?: StyleProp<ViewStyle>;
+}
+
+const ListItem: React.FC<IListItemProps> = ({
+  children,
+  leftContent,
+  rightContent,
+  onPress,
+  size = 'medium',
+  selected,
+  disabled,
+  divider,
+  testID,
+  style,
+}) => {
+  const styles = useStyles();
+
+  const handlePress = (event: GestureResponderEvent) => {
+    if (!disabled && onPress) {
```

Comment:
Why do we need to check `disabled`. `handlePress` will never be called in this case.